### DISCUSSION TITLE
windows user

### DISCUSSION THREAD

**Discussion by Chadnon** [achannarasappa/ticker#167] [Q&A]
How do Windows users run programs? I ran main.exe on Windows and got no results.

**Comment by achannarasappa** [comment]
Since ticker is a terminal application, you'll have to open it through PowerShell on Windows e.g.: `PS C:\Users\ani\Desktop> .\ticker.exe -w TSLA`

Hope that helps

**Reply by Chadnon** [reply]
Thanks!
